app/src/cli/mod.rs

```diff
@@ -70,6 +70,20 @@ pub enum Commands {
     },
 }
 
+#[derive(Subcommand, Clone, Copy, Debug)]
+pub enum GoldenImageCommand {
+    /// Renders an image and saves it after waiting for the specified number of seconds
+    GoldenImageUpdate {
```

Comment:
extremely minor nit: can drop the `GoldenImage` in the name of these variants, it's implied by being part of `GoldenImageCommand`